projects/ad2s1210_iio/README.txt

```diff
@@ -0,0 +1,43 @@
+Evaluation Boards/Products Supported
+------------------------------------
+EVAL-AD2S1210
+
+
+Overview
+--------
+This is an IIO firmware application to evaluate the AD2S1210 device.
+This code was developed and tested on SDP-K1 controller board: https://os.mbed.com/platforms/SDP_K1/
+Use of the Mbed platform allows code to be ported to other Mbed supported target boards with little or no modifications.
+
+Product details: AD2S1210.
+Product Evaluation board details: EVAL-AD2S1210SDZ
```

Comment:
Do we have an arduino based board for this part? 
I see that the macros in the app_config_mbed.h maps the pins to arduino headers. So would the board ID here need an update?

Reply:
With current hardware available the setup is fly wires from the arduino pins on the sdp-k1 to a breakout board that is connected to the ad2s1210 eval. I need to describe this better on the readme. will do.